modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/EpisodeDownloadError.kt

```diff
@@ -54,29 +55,29 @@ data class EpisodeDownloadError(
         const val PODCAST_UUID_KEY = "podcast_uuid"
         const val TASK_DURATION_KEY = "duration"
         const val HTTP_STATUS_CODE_KEY = "http_status_code"
-        const val CONTENT_TYPE_KEY = "content_type"
+        const val CONTENT_TYPE_KEY = "http_content_type"
```

Comment:
Just to double check do we also want to update this key, right?

Reply:
I had this PR on auto-merge so it merged before I responded here. 😅 

Bytes yes, this key was supposed to be updated because iOS uses `content_type` for something else in all events.